src/main/java/org/example/controller/RestController.java

```diff
@@ -0,0 +1,55 @@
+package org.example.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.example.model.User;
+import org.example.service.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@org.springframework.web.bind.annotation.RestController
```

Comment:
N: Naming of the class is not appropriate.
UserController.
so we won't need it (full import of this annotation)